utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+            os.makedirs(output_dir, exist_ok=True)
+            
+            # First, get the row count from database
+            db_row_count = self.get_table_row_count(table_name, schema)
+            result['db_count'] = db_row_count
+            
+            if db_row_count == -1:
+                result['error'] = "Failed to get row count from database"
+                return result
+            
+            if db_row_count == 0:
+                logger.warning(f"Table {schema}.{table_name} is empty (0 rows)")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)